### DISCUSSION TITLE
Direction of Lazer

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#25729] [Q&A]
I'm seeing a rise in changes that're objectively bad in the name of having greater community acceptance (what really irked me and pushed me over was seeing #25594). It sounds like you guys are prioritizing community acceptance above all, sacrificing the vision was held for this game (either by peppy or the osu! team) as a consequence.

If this is all temporary and will be reverted after ranked play, then please say so (tho even then that's a lot of work wasted). Otherwise, I don't get why you guys are doing this. If you have a confident vision in the game, the community will follow, there's no need to sacrifice in order to get more players to switch over. We saw this happen with old vs new web. Over time, as the player base increased, more people stuck with new, with old eventually only being used for some legacy features like apiv1 (which has been ported over now). Why can't the same be done for Lazer?

It also seems like you guys are placing an insane weight on the top player feedback server, which is going to be naturally biased towards stable due to just how much time they've spent on it in order to get good in the first place. Not to say that having their opinion is bad, but it seems like ur weighing theirs way too heavily.

If any of this is wrong, please let me know. It pains me to see such an amazing project be ruined by mob mentality and community Stockholm syndrome.

**Comment by bdach** [comment]
> If you have a confident vision in the game, the community will follow, there's no need to sacrifice in order to get more players to switch over.

Will it though?

lazer has been the butt of jokes for years now for authoritative decisions the dev team has taken, many of which have since been reverted. You may say some of them were "objectively" terrible ideas, but how do we tell? For instance, VINXIS has been railing against slider head accuracy since _2020_ (can't find sources for this other than pp server history because anything else probably would have been on twitter and that website is now unbrowseable). It's been brought up again [a year ago](https://github.com/ppy/osu/discussions/17342). Searching "slider acc" turns up [this reddit post](https://www.reddit.com/r/osugame/comments/159vicd/i_hope_lazer_never_comes_out/). So why _wouldn't_ we revert slider heads to behave more like stable?

I'm sorry but you are a single data point here. Your opinion is noted but we have to ship something at some point. I tire of these constant back-and-forths over every single mechanic. Maybe the changes from lazer can be revisited at a later point, I dunno. But right now, we're trying to cut scope of change to have something to finally ship. What we have _now_, after significant concessions, is difficult enough to get out the door.

**Reply by bdach** [reply]
Also on this:

> It also seems like you guys are placing an insane weight on the top player feedback server

Yes and no? Yes in that they have been the primary people we have been listening to for the past few months, because every single poll conducted before had people coming in saying "you're asking the questions wrong". But also no, because the feedback from the server in question was really not... there, in my impression. There was very little of it. So I don't know that they influenced the decision making _that_ much. Most things that were reverted thus far were the dev team's shot calls one way or another.

**Reply by Rekunan** [reply]
That sentence was based on my memory of how the community reacted to new web, and how ppl naturally gravitate towards confidence. To talk on the slideracc instance, I'll bring up how tournaments have been handling them perfectly fine.

If the changes can be revisited, then that'd be great, tho I do understand that's potentially a lot of work wasted. And I also understand how you guys want something shipped, so if I do become more active, I'll only focus on what hasn't been pushed yet like that open pr. Part of me wishes I could've posted earlier, but it's just a natural consequence of my time.

On the top player server, that's rly interesting to hear, thanks for the insight on that, I'm glad my assumptions on it and the dev team were wrong.

**Comment by peppy** [comment]
If you have been following along even a bit, you'd see me mentioning precisely this kind of thing (lack of feedback from the larger player base, aka the non-top players which matter more to me).

If you have an issue with the PR you linked, how about providing feedback in it? Be the change you want to see.

> It pains me to see such an amazing project be ruined by mob mentality and community Stockholm syndrome.

This screams failing to understand the issues at hand. Maybe you should read up on why the slider changes are happening in the first place.

**Reply by Rekunan** [reply]
That's good to hear, then. I haven't been as active as I would've liked before, but I'll def be trying to do that more with the time I have now. Starting with yeah, reading more into detail on that pr.
